scripts/apps/helpers.js

```diff
@@ -135,6 +135,29 @@ export function parseSeparator(string){
     return string.split(sep).map(n => n.trim())
 }
 
+// Helper method to turn a search string into one or more regular expressions.
+export function compilePossibilities(string) {
```

Comment:
*technically* we could speed up stuff by caching the compiled regex's, but I don't think it would be needed unless someone was doing something very, very odd.